bodhi/tests/server/services/test_updates.py

```diff
@@ -599,6 +601,195 @@ def test_provenpackager_request_privs(self, publish, *args):
         res = app.get('/updates/%s' % str(nvr), status=200)
         eq_(res.json_body['can_edit'], False)
 
+    @mock.patch(**mock_taskotron_results)
+    @mock.patch(**mock_valid_requirements)
+    @mock.patch('bodhi.server.notifications.publish')
+    @mock.patch.dict('bodhi.server.config.config', {'ci.required': True})
+    def test_provenpackager_request_queued_in_ci(self, publish, *args):
+        "Ensure provenpackagers can change the request for any update"
```

Comment:
Can we convert this to a docblock instead of a normal string (triple quotes)? Also, the name of the test doesn't seem to correlate with this description.